src/main/java/ua/softserveinc/tc/service/impl/BookingServiceImpl.java

```diff
@@ -164,18 +164,40 @@ public Boolean checkForDuplicateBookingSingle(BookingDto bookingDto) {
         return isDuplicate;
     }
 
-
+    /**
+     * The method finds out is there available space in
+     * the rooms for given listDTO.
+     * @param listDTO list of BookingDto
+     * @return true if there is available places in the room
+     */
+    private boolean isAvailabilePlacesInTheRoom(List<BookingDto> listDTO) {
+        int availablePlaces = 0;
+        int needPlaces = 1;
+        Date theSameDay = null;
+        for(BookingDto bdto : listDTO) {
+            if(bdto.getDateStartTime().equals(theSameDay)) {
```

Comment:
why you compare with null using method equal? it will be always return false and it is not good practice